spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -499,29 +499,132 @@ the method finishes and the `end = true` element on the confirmTrip method force
 [[compensating-activities]]
 ==== Compensating Activities
 
+As remarked elsewhere, the LRA specification attempts to enforce some of the traditional
+guarantees provided by transactional systems such as atomicity (all or nothing)
+whilst relaxing others, such as isolation of work amongst participants. The characteristic
+of strong consistency of data (the system can only be observed to transition between
+consistent states) is also relaxed in favour of what is referred to as `eventual
+consistency`. The LRA specification ensures atomicity and eventual consistency by
+placing certain requirements on the entities that participate in the protocol
+which we now discuss (further details of these responsibilities can be found in
+the javadoc for the participant annotations).
+
 The application developer indicates which method to use for a compensating
-activity by annotating it with the `@Compensate` annotation.
+activity by marking it with the `@Compensate` annotation.
 Whenever the associated resource is invoked in the context of
-an LRA the method corresponding to the `@Compensate` method MUST be enlisted with
-the LRA. If the LRA is subsequently cancelled then the compensation method MUST be invoked.
+an LRA the method corresponding to this `@Compensate` method MUST be enlisted with
+the LRA: enlistment means that if the LRA is subsequently cancelled then the compensation
+method MUST be invoked.
 
 The specification does not mandate when this method is invoked
-but it does guarantee that it will eventually be called. Under failure
-conditions the system will keep retrying until it is certain that the
+but it does guarantee that it will eventually be called (this is the precise
+meaning of the term `eventual consistency` as used in this specification).
+Under failure conditions the system will keep retrying until it is certain that the
 participant has been successfully notified.
 
-An example of how to use this annoation is:
+The LRA model is quite flexible in how business methods perform compensations.
+If a compensating activity is brief then the synchronous model may be appropriate:
 
 [source,java]
 ----
-  @PUT
```

Comment:
Leave the `@PUT` annotation please. The example seems incomplete from JAX-RS perspective when only `@Path` is used on method. (also in other examples please)